api/interfaces/interfaces.go

```diff
@@ -18,11 +20,21 @@ type SignUpForm struct {
 	Password string `json:"password"`
 }
 
+type LogInForm struct {
+	Email    string `json:"email"`
+	Password string `json:"password"`
+}
+
 type User struct {
+	Id         primitive.ObjectID   `json:"_id"       bson:"_id"`
 	Username   string               `json:"username"      bson:"username"`
 	Email      string               `json:"email"     bson:"email"`
 	Password   string               `json:"password"  bson:"password"`
 	Items      []interface{}        `json:"items"     bson:"items"`
 	Loomies    []primitive.ObjectID `json:"loomies"   bson:"loomies"`
 	IsVerified bool                 `json:"isVerified"   bson:"isVerified"`
 }
+
+type TokenInfo struct {
```

Comment:
Can you also include the username?

Reply:
Create a new type of user, one for the creation and another for log in because if I use the same and when adding the id to the database it is inserted as 00...

Reply:
I meant to just add the Username field in the TokenInfo struct.